docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -95,6 +95,7 @@ Alpha features might be changed or removed without prior notice.
 | `authnService`                     | Use new auth service to perform authentication                                                                                                                               |
 | `sessionRemoteCache`               | Enable using remote cache for user sessions                                                                                                                                  |
 | `alertingBacktesting`              | Rule backtesting API for alerting                                                                                                                                            |
+| `individualCookiePreferences`      | Support overriding cookie preferences on an individual level                                                                                                                 |
```

Comment:
what does 'on an individual level' mean?

Reply:
user, should I substitute individual for user?